### ISSUE TITLE
`More commands` three dots button on toolbar does not reflect status

### ISSUE THREAD

**Issue by krassowski** [jupyterlab/jupyterlab#14464]
## Description

When screen space is tight the toolbar items get collapse in a second, hidden row under "More commands" button (`...`). This button does indicate whether it is pressed or not - it always has `aria-pressed="false"`

## Reproduce

1. Open notebook
2. resize until toolbar overflow
3. Press `...`
4. See the `...` button does not change

Not pressed:

![Screenshot from 2023-04-30 12-13-40](https://user-images.githubusercontent.com/5832902/235349947-55d9a777-b0d4-417c-b94b-fa9a8a4988ae.png)

Pressed:

![Screenshot from 2023-04-30 12-13-37](https://user-images.githubusercontent.com/5832902/235349946-8bdd0b18-9d66-4d05-93e4-c9bc95310ab4.png)

## Expected behavior

we reflect the state by updating
- `aria-pressed="true"`
- some visual style representing "pressed" (e.g. background)

## Context

- JupyterLab version: 4.0.0rc0 and earlier

**Comment by xgdgsc** [comment]
Sometimes I get an empty long toolbar that has this "more commands" button at the end and after pressing it shows the normal icons under the empty long toolbar. Resizing the window make it return to normal.

**Comment by xgdgsc** [comment]
Yes you can open an issue. It happens to me also after a refresh like that issue.

**Comment by MUFFANUJ** [comment]
working on it will, soon open a PR
